### PR TITLE
Some projects from https://github.com/topics/coq\?o\=desc\&s\=stars and other prominent projects

### PR DESCRIPTION
**Pull request by anton-trunov**
Would be nice to also add these:
- https://github.com/MetaCoq/metacoq
- https://github.com/jwiegley/category-theory.

Need some categories for that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +61,7 @@ Is your favorite project not listed? Fork and [create a Pull Request](https://gi
 - [ProofGeneral](https://proofgeneral.github.io/) — Generic interface for proof assistants based on the extensible, customizable text editor Emacs.
 - [Company-Coq](https://github.com/cpitclaudel/company-coq) — IDE extensions for Proof General's Coq mode.
 - [VSCoq](https://github.com/coq-community/vscoq) – Visual Studio Code extension for Coq.
+- [jsCoq](https://github.com/ejgallego/jscoq) - A port of Coq to Javascript -- Run Coq in your Browser.
```
Comment (palmskog) [inline README.md]:
```suggestion
- [jsCoq](https://github.com/ejgallego/jscoq) — Port of Coq to JavaScript which enables running Coq projects in a browser.
```

File: README.md [hunk 2]
```diff
@@ -78,20 +84,26 @@ Is your favorite project not listed? Fork and [create a Pull Request](https://gi
 ## Plugins
 
 - [CoqHammer](https://github.com/lukaszcz/coqhammer) – General-purpose automated reasoning hammer tool for Coq that combines learning from previous proofs with the translation of problems to the logics of automated systems and the reconstruction of successfully found proofs.
+- [Equations](https://github.com/mattam82/Coq-Equations) - A function definition package for Coq.
+- [QuickChick](https://github.com/QuickChick/QuickChick) - Randomized Property-Based Testing Plugin for Coq.
 
 ## Tools
 
 - [Ott](https://github.com/ott-lang/ott) – Tool for writing definitions of programming languages and calculi that can be exported to Coq.
+- [Cosette](https://github.com/uwdb/Cosette) - Cosette is an automated SQL solver.
```
Comment (palmskog) [inline README.md]:
```suggestion
- [Cosette](https://github.com/uwdb/Cosette) – Automated solver for reasoning about SQL equivalences.
```

File: README.md [hunk 3]
```diff
@@ -78,20 +84,26 @@ Is your favorite project not listed? Fork and [create a Pull Request](https://gi
...
 
 ## Type Theory and Mathematics
 
 - [Four Color Theorem](https://github.com/math-comp/fourcolor) – Formal proof of the Four Color Theorem, a landmark result of graph theory.
 - [Homotopy Type Theory](https://github.com/HoTT/HoTT) – Development of homotopy-theoretic ideas in Coq.
 - [Odd Order Theorem](https://github.com/math-comp/odd-order) – Formal proof of the Odd Order Theorem, a landmark result of finite group theory.
+- [UniMath](https://github.com/UniMath/UniMath) -- This coq library aims to formalize a substantial body of mathematics using the univalent point of view.
```
Comment (palmskog) [inline README.md]:
```suggestion
- [UniMath](https://github.com/UniMath/UniMath) – Library which aims to formalize a substantial body of mathematics using the univalent point of view.
```

File: README.md [hunk 4]
```diff
@@ -78,20 +84,26 @@ Is your favorite project not listed? Fork and [create a Pull Request](https://gi
...
 
 ## Verified Software
 
 - [CompCert](http://compcert.inria.fr) – High-assurance compiler for almost all of the C language (ISO C99), generating efficient code for the PowerPC, ARM, RISC-V and x86 processors.
+- [Verdi](https://github.com/uwplse/verdi) - A framework for formally verifying distributed systems implementations in Coq.
+- [Verdi Raft](https://github.com/uwplse/verdi-raft) - An implementation of the Raft distributed consensus protocol, verified in Coq using the Verdi framework.
```
Comment (palmskog) [inline README.md]:
```suggestion
- [Verdi Raft](https://github.com/uwplse/verdi-raft) - Implementation of the Raft distributed consensus protocol, verified in Coq using the Verdi framework.
```

File: README.md [hunk 5]
```diff
@@ -78,20 +84,26 @@ Is your favorite project not listed? Fork and [create a Pull Request](https://gi
...
 
 - [Four Color Theorem](https://github.com/math-comp/fourcolor) – Formal proof of the Four Color Theorem, a landmark result of graph theory.
 - [Homotopy Type Theory](https://github.com/HoTT/HoTT) – Development of homotopy-theoretic ideas in Coq.
 - [Odd Order Theorem](https://github.com/math-comp/odd-order) – Formal proof of the Odd Order Theorem, a landmark result of finite group theory.
+- [UniMath](https://github.com/UniMath/UniMath) -- This coq library aims to formalize a substantial body of mathematics using the univalent point of view.
 
 ## Verified Software
 
 - [CompCert](http://compcert.inria.fr) – High-assurance compiler for almost all of the C language (ISO C99), generating efficient code for the PowerPC, ARM, RISC-V and x86 processors.
+- [Verdi](https://github.com/uwplse/verdi) - A framework for formally verifying distributed systems implementations in Coq.
```
Comment (palmskog) [inline README.md]: Let's put this in a "Framework" section instead, along with, potentially, Iris, VST, Fiat

File: README.md [hunk 6]
```diff
@@ -78,20 +84,26 @@ Is your favorite project not listed? Fork and [create a Pull Request](https://gi
 ## Plugins
 
 - [CoqHammer](https://github.com/lukaszcz/coqhammer) – General-purpose automated reasoning hammer tool for Coq that combines learning from previous proofs with the translation of problems to the logics of automated systems and the reconstruction of successfully found proofs.
+- [Equations](https://github.com/mattam82/Coq-Equations) - A function definition package for Coq.
```
Comment (palmskog) [inline README.md]:
```suggestion
- [Equations](https://github.com/mattam82/Coq-Equations) – Function definition package for Coq.
```

File: README.md [hunk 7]
```diff
@@ -78,20 +84,26 @@ Is your favorite project not listed? Fork and [create a Pull Request](https://gi
...
+- [QuickChick](https://github.com/QuickChick/QuickChick) - Randomized Property-Based Testing Plugin for Coq.
```
Comment (palmskog) [inline README.md]:
```suggestion
- [QuickChick](https://github.com/QuickChick/QuickChick) – Plugin for randomized property-based testing.
```

File: README.md [hunk 8]
```diff
@@ -39,6 +39,11 @@ Is your favorite project not listed? Fork and [create a Pull Request](https://gi
 - [Joachim Breitner's Coq posts](http://www.joachim-breitner.de/blog/tag/Coq)
 - [Gregory Malecha's blog](https://gmalecha.github.io)
 - [Coq Exchange: ideas and experiment reports about Coq](https://project.inria.fr/coqexchange/news/)
+
+## Tutorials, cheatsheets
```
Comment (palmskog) [inline README.md]:
```suggestion
## Tutorials
```
Please add this to the table of contents. Maybe "Tutorials and Hints" is better? Anchor becomes `#tutorials-and-hints`

**Comment by palmskog** [comment]
Category theory fits under "Type Theory and Mathematics", MetaCoq fits either under plugin or library, I'd say more towards plugin.

**Comment by anton-trunov** [comment]
Great suggestions! Done.

**Comment by anton-trunov** [comment]
We should probably cover these https://deepspec.org/page/Project/

**Pull request merged**